src/amplitude-wrapper.js

```diff
@@ -228,6 +232,7 @@ const LOG_PREFIX = '[Amplitude / GTM]';
       
       const userAgentEnrichmentOptions = configuration['userAgentEnrichmentOptions'];
       const pageViewLegacy = configuration['pageViewLegacy'];
+      const sessionReplayPlugin = configuration['sessionReplayPlugin'];
```

Comment:
Can we rename it to `sessionReplay` or `enableSessionReplay` to follow the naming patterns, for example, config.autocapture.sessions, config.autocapture.elementInteractions, etc? I'm more into the former.